### PR TITLE
gazebo内で、逆運動学でfetchを動かす。

### PR DESCRIPTION
**Pull request by 708yamaguchi**

### CODE REVIEW

File: jsk_fetch_robot/README.md [hunk 1]
```diff
@@ -119,7 +119,32 @@ fetch-interface function APIs
   ```
 (send *ri* :stop-grasp)
   ```
+  
+gazebo内で、逆運動学でfetchを動かす。
```
Comment (furushchev) [inline jsk_fetch_robot/README.md]: @708yamaguchi Hi, thank you for great contribution. Could you please re-write explanation in English?

Comment (708yamaguchi) [inline jsk_fetch_robot/README.md]: @furushchev Thank you for your advice. I rewrote my comment.

Comment (furushchev) [inline jsk_fetch_robot/README.md]: @708yamaguchi OK, next please could you make it one commit by `git rebase`, and please add clear commit message (see https://github.com/jsk-ros-pkg/jsk_common/blob/master/CONTRIBUTION.md and commit messages at other pull requests)

Comment (708yamaguchi) [inline jsk_fetch_robot/README.md]: @furushchev  rebase しました。

**Comment by knorth55** [comment]
@708yamaguchi 
you need to resolve conflict.
if you can't do it by yourself, some expert in JSK may help you :)

```
roscd jsk_fetch_robot
git fetch --all

# git rebase start
git rebase origin/master
## fix conflict 
## after fixing conflict
git add README.md
git rebase --continue
# git rebase end

# check your commit
git show
# push to your branch
git push 708yamaguchi master
```

**Comment by 708yamaguchi** [comment]
@knorth55 
I resolved the conflict, but there might be some mistakes...

File: jsk_fetch_robot/README.md [hunk 2]
```diff
@@ -114,12 +127,46 @@ fetch-interface function APIs
   ```
 (send *ri* :start-grasp)
   ```
+
```
Comment (knorth55) [inline jsk_fetch_robot/README.md]: tailing white space

File: jsk_fetch_robot/README.md [hunk 3]
```diff
@@ -114,12 +127,46 @@ fetch-interface function APIs
...
+ move to point (0 8150 0) [mm], based on absolute map positions (CAUTION, this will move the real robot)
+ ```
+ (send *ri* :move-to (make-coords :pos #f(0 8150 0)) :frame-id "/map") ;; :retry 1 ;;[mm]
+ ```
+ 
+- speak
+ 
+ ```
+  (send *ri* :speak "hello")
+ ```
```
Comment (knorth55) [inline jsk_fetch_robot/README.md]: many tailing white spaces
tailspace

**Comment by knorth55** [comment]
@708yamaguchi 
I have no idea you solved conflict correctly, but it seems that there is no weird point.
If this PR contain all changes you wanted to make, you solved conflict correctly!
One small review is that there are many tailing white spaces, which is not smart for code format.
Please clean up theses spaces, and your PR would be great!

**Comment by 708yamaguchi** [comment]
move to #726

**Pull request closed without merging**